src/test/resources/test-rom/spec/programs/alias_spec.lua

```diff
@@ -0,0 +1,33 @@
+local capture = require "test_helpers".capture_program
+
+describe("The alias program", function()
+
+    it("displays its usage when given to many arguments", function()
+        
+        expect(capture(stub, "alias a b c"))
+            :matches { ok = true, output = "Usage: alias <alias> <program>\n", error = "" }
+    end)
+
+    it("list alias", function()
+        
+        expect(capture(stub, "alias"))
+            :matches { ok = true, output = "\n\n\n", error = "" }
```

Comment:
The fact that `alias` isn't emitting anything makes me suspect that `capture` isn't handling anything correctly. That's definitely not the expected behaviour.

Reply:
capture has problems with the textutils functions. hello.lua will crash, if you run it with capture.